examples/Recommendations/DetectAndApplyRecommendations.php

```diff
@@ -124,57 +121,109 @@ public static function runExample(GoogleAdsClient $googleAdsClient, int $custome
+                $recommendation->getResourceName(),
+                $recommendation->getCampaign(),
+                PHP_EOL
             );
-
-            // Creates apply operations for all the recommendations found.
-            $applyRecommendationOperations = [];
-            foreach ($stream->iterateAllElements() as $googleAdsRow) {
-                /** @var GoogleAdsRow $googleAdsRow */
-                $applyRecommendationOperations[] = new ApplyRecommendationOperation([
-                    'resource_name' => $googleAdsRow->getRecommendation()->getResourceName()
-                ]);
-            }
-
-            if ($applyRecommendationOperations) {
-                // Sends the apply recommendation request and prints information.
-                $response = $recommendationServiceClient->applyRecommendation(
-                    $customerId,
-                    $applyRecommendationOperations
-                );
-                foreach ($response->getResults() as $result) {
-                    /** @var ApplyRecommendationResult $result */
-                    printf(
-                        'Applied recommendation with resource name: "%s".%s',
-                        $result->getResourceName(),
-                        PHP_EOL
-                    );
-                }
-            } else {
-                print 'No recommendations were found.' . PHP_EOL;
-            }
-            print PHP_EOL;
-
-            if ($i < self::NUMBER_OF_RUNS - 1) {
+            if (!is_null($recommendation->getKeywordRecommendation())) {
+                $keyword = $recommendation->getKeywordRecommendation()->getKeyword();
                 printf(
-                    "Waiting %d seconds before checking for additional recommendations.%s",
-                    self::PERIOD_IN_SECONDS,
+                    "\tKeyword = '%s'%s\ttype = '%s'%s",
+                    $keyword->getText(),
+                    PHP_EOL,
+                    KeywordMatchType::name($keyword->getMatchType()),
                     PHP_EOL
                 );
-                sleep(self::PERIOD_IN_SECONDS);
             }
+            // Creates an ApplyRecommendationOperation that will be used to apply this
+            // recommendation, and add it to the list of operations.
+            $operations[] = self::buildRecommendationOperation($recommendation->getResourceName());
+        }
+        // If there are operations present, send a request to apply the recommendations.
+        if (!empty($operations)) {
+            self::applyRecommendations($googleAdsClient, $customerId, $operations);
+        }
+    }
+
+    /**
+     * Creates a ApplyRecommendationOperation to apply the given recommendation.
+     *
+     * @param string $recommendationResourceName the resource name of the recommendation to apply
+     * @return ApplyRecommendationOperation the created ApplyRecommendationOperation
+     */
+    private static function buildRecommendationOperation(
```

Comment:
Per the Perl changes, can you add `build_apply_recommendation_operation` tags around this example? See https://github.com/googleads/google-ads-perl/pull/124/files